src/shared/db.py

```diff
@@ -219,6 +238,19 @@ def get_user(self, uid: str) -> Optional[SubHubDeletedAccountModel]:
             logger.error("get user", uid=uid)
             return None
 
+    def find_by_uid(self, uid: str):
+        items = [item for item in self.model.query(uid)]
+        return items
+
+    def find_by_cust(self, customer_id: str) -> Optional[SubHubDeletedAccountModel]:
+        logger.info("find cust start", customer_id=customer_id)
+        for item in self.model.cust_index.query(customer_id):
+            # for item in self.model.query("user_id", self.model.cust_id == customer_id):
+            logger.info("find cust", item=item, user_id=item.user_id)
+            return item
+        logger.info("find cust did not find any results")
```

Comment:
this message is not very helpful if it doesn't mention what it was searching for

Reply:
You're right, removed.